### PR TITLE
Add form with meeting time and duration input fields and Ad-lib content to UI

### PR DESCRIPTION
**Pull request by dhruvi618**
Next steps: Add more input fields to UI and retrieve form inputs on submit

### CODE REVIEW

File: frontend/src/menu-bar.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  title: {
+    flexGrow: 1
+  }
+}));
+
+// Create menu bar component with title and help icon (with About and FAQ links)
+export default function MenuAppBar() {
+  const classes = useStyles();
+  const [anchorElement, setAnchorElement] = React.useState(null);
+  const open = Boolean(anchorElement);
```
Comment (kmh287) [inline frontend/src/menu-bar.js:27]: Probably a react question but why does this need to be wrapped in a Boolean? Could it be used directly, or is this used for a truthy --> boolean conversion?

File: frontend/yarn.lock [hunk 2]
```diff
@@ -1069,7 +1069,7 @@
   dependencies:
     regenerator-runtime "^0.13.4"
 
-"@babel/runtime@^7.0.0", "@babel/runtime@^7.10.2", "@babel/runtime@^7.3.4", "@babel/runtime@^7.4.5", "@babel/runtime@^7.5.1", "@babel/runtime@^7.7.2", "@babel/runtime@^7.8.4":
+"@babel/runtime@^7.0.0", "@babel/runtime@^7.10.2", "@babel/runtime@^7.3.1", "@babel/runtime@^7.3.4", "@babel/runtime@^7.4.4", "@babel/runtime@^7.4.5", "@babel/runtime@^7.5.1", "@babel/runtime@^7.5.5", "@babel/runtime@^7.7.2", "@babel/runtime@^7.8.3", "@babel/runtime@^7.8.4", "@babel/runtime@^7.8.7":
```
Comment (tcwanggoogle) [inline frontend/yarn.lock:1072]: Question: What causes this file to change? (just curious if we'll see this file change on every PR, or not)

File: frontend/src/menu-bar.js (hunk 1 above)
Comment (dhruvi618) [reply frontend/src/menu-bar.js:27]: I realized I misinterpreted our original UI design for the menubar (was not supposed to have two options, just a click that redirects it to a FAQ page) so I got rid of this code, but this line could be use directly without the Boolean wrapper.

**Comment by dhruvi618** [comment]
UI Design: https://screenshot.googleplex.com/Ra24iNGW7G1

File: frontend/yarn.lock (hunk 2 above)
Comment (dhruvi618) [reply frontend/yarn.lock:1072]: This file changes when dependencies/libraries are added or updated. For example, in this PR, I added a material-ui icons import (to use the help icon) so it caused updates in the file. I don't think it should happen in every PR.

**Pull request merged**